lib/base.js

```diff
@@ -47,7 +47,7 @@ class BaseModel {
             }
         };
 
-        return nodeify(this.datastore.get, config, callback);
+        return nodeify(this.datastore, this.datastore.get, config, callback);
```

Comment:
What if we did something like  `return nodeify(this.datastore, 'get', config, callback); ` instead? Then we don't need to repeat the context twice.